static/app/components/organizations/projectPageFilter/index.tsx

```diff
@@ -159,6 +169,7 @@ export function ProjectPageFilter({
       });
     },
     [
+      pageFilterValue,
```

Comment:
Looks like you're missing `value` in this dep array?

Reply:
Oh true just added it in https://github.com/getsentry/sentry/pull/48491/commits/3288695098c957035bc4e319028e7de34b73abb7. Also I added some tests in https://github.com/getsentry/sentry/pull/48491/commits/1dcfb847acca77eff5bb0174c25c0df0000ed6d0